examples/options/options-trade-options-with-alpaca.ipynb

```diff
@@ -0,0 +1,807 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> <b>Note: This Jupyter Notebook is associated with the article: [How To Trade Options with Alpaca's Dashboard and Trading API](https://alpaca.markets/learn/how-to-trade-options-with-alpaca).</b>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/alpacahq/alpaca-py/blob/master/examples/options/trade-options-with-alpaca.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Options Trading Using Alpaca-py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# 1.1 Install / Update Alpaca-py\n",
+    "!pip install --upgrade alpaca-py"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# 1.2 Import Necessary Packages\n",
+    "import json\n",
+    "from datetime import datetime, timedelta\n",
+    "from zoneinfo import ZoneInfo\n",
+    "import pandas as pd\n",
+    "\n",
+    "from alpaca.trading.client import TradingClient\n",
+    "from alpaca.data.timeframe import TimeFrame, TimeFrameUnit\n",
+    "from alpaca.data.historical.stock import StockHistoricalDataClient, StockLatestTradeRequest\n",
+    "from alpaca.data.historical.option import OptionHistoricalDataClient\n",
+    "from alpaca.trading.stream import TradingStream\n",
+    "from alpaca.data.live.option import OptionDataStream\n",
+    "\n",
+    "from alpaca.data.requests import (\n",
+    "    OptionBarsRequest,\n",
+    "    OptionTradesRequest,\n",
+    "    OptionLatestQuoteRequest,\n",
+    "    OptionLatestTradeRequest,\n",
+    "    OptionSnapshotRequest,\n",
+    "    OptionChainRequest\n",
+    ")\n",
+    "from alpaca.trading.requests import (\n",
+    "    GetOptionContractsRequest,\n",
+    "    GetAssetsRequest,\n",
+    "    MarketOrderRequest,\n",
+    "    GetOrdersRequest,\n",
+    "    ClosePositionRequest,\n",
+    "    OptionLegRequest\n",
+    ")\n",
+    "from alpaca.trading.enums import (\n",
+    "    AssetStatus,\n",
+    "    ExerciseStyle,\n",
+    "    OrderSide,\n",
+    "    OrderClass,\n",
+    "    OrderType,\n",
+    "    TimeInForce,\n",
+    "    QueryOrderStatus,\n",
```

Comment:
can probably move all constants like those here and `PAPER`, etc to their own cell above and say something like ```# Constants required for the proper functioning of this notebook```